docs/resources/event_gateway_consume_policy_decrypt.md

```diff
@@ -1,130 +0,0 @@
----
-# generated by https://github.com/hashicorp/terraform-plugin-docs
-page_title: "konnect_event_gateway_consume_policy_decrypt Resource - terraform-provider-konnect-beta"
```

Comment:
These look like breaking changes - are we generating from latest platform-api `main` branch?

Reply:
This should push the latest changes, but if you're afraid that's not it I can regenerate.

Reply:
Yep, looks like some changes have been made to the Event Gateway Spec on platform-api which broke tooling that operates on it. I'll need to check how to fix it, and send a PR.
@slonka by when do you want this merged and released? Is ~Tuesday fine?

Reply:
> @slonka by when do you want this merged and released? Is ~Tuesday fine?

@harshadixit12 need to check with the customer but I think it should be fine, thank you 🙇

Reply:
@harshadixit12 pulled main, let me know if this works